packages/cli/config/jest.js

```diff
@@ -242,7 +231,7 @@ async function getRootConfig() {
   }
```

Comment:
Just above here is the logic for whether we're running in multi-project mode or just loading a single jest config file